.github/workflows/build.yml

```diff
@@ -55,6 +55,14 @@ jobs:
         run: |
           npm run build
 
+      - name: Diff Tokens
```

Comment:
Not sure why we're having to manually download that file and diff against it. Doesn't git already know what changed. For example, wouldn't something like `git diff ./tokens/css/variables.css` get the diff assuming that file changed ?

Reply:
This file is generated as part of the build process. Basically, we want to make it visible on the PR how the generated file is changing.